RELEASE.md

```diff
@@ -1,33 +1,54 @@
-## Making a release of the Checker Framework Gradle plugin
+## Making and merging pull requests
 
-This document describes how to make a release of the Checker Framework
-Gradle plugin, which this repository contains.
+The Checker Framework Gradle plugin (which this repository contains)
+uses a continuous-delivery style of releases: any time that the
+behavior of the plugin changes, a new release is cut.
 
-### Prequisites
+### Making a pull request
+
+All pull requests should be made from a feature branch. Never push
+directly to `master`.
+
+When you make a pull request that will change any behavior
+of the plugin (including bug fixes and new features), update the version string on 
+your feature branch using these steps:
+1. Make the changes you wish to merge into the plugin.
+2. Choose a new version string. Please try to respect
+[semantic versioning](https://semver.org/). Do not augment the major
+version without explicit approval from all the maintainers.
+3. Change the version of the plugin. Search for the current plugin version
+in the directory containing this file, and replace all instances of it
+with the new version string. Commit the result.
+4. Push your feature branch and make a pull request against the `master` branch.
```

Comment:
It's easier to follow instructions that put each action in its own step.  Split this into two.